docs/features/feature-guides/compliance-forms/create-a-form.md

```diff
@@ -0,0 +1,159 @@
+---
+title: Create a Form
+---
+
+import FeatureAvailability from '@site/src/components/FeatureAvailability';
+
+# Create a DataHub Compliance Form
+<FeatureAvailability/>
+
+This guide will walk you through creating and assigning Compliance Forms, including:
+
+1. Defining your Compliance Form
+2. Creating Questions to be completed by assignees
+3. Selecting the in-scope Assets for the Compliance Form
+4. Assigning Forms to specific Users
+5. Publishing your Form
+
+:::note
+Creating and managing Compliance Forms via the UI is only available in DataHub Cloud. If you are deployed with DataHub Core, please see the [Compliance Forms API Guide](../../../api/tutorials/forms.md).
+:::
+
+## Create and Assign a Compliance Form from the DataHub Cloud UI
+
+### Prerequisites
+
+In order to create or edit a Compliance Form, your DataHub User must have the **Manage Compliance Forms** permission.
```

Comment:
permission -> privilege.

Usually we are also specific between privileges of type **Metadata** and of Type **Platform**. 

Suggestion:

In order to create, edit, or remove Compliance Forms, you must have the **Manage Compliance Forms** platform privilege.